src/core/Akka.Streams/Util/EnumeratorEnumerable.cs

```diff
@@ -60,10 +60,7 @@ public ContinuallyEnumerator(Func<IEnumerator<T>> enumeratorFactory)
            
             public void Dispose() => _enumeratorFactory = null;
 
-            /// <inheritdoc/>
-            /// <exception cref="ArgumentException">
```

Comment:
Did this also need to be removed @eaba ?

Reply:
DocFx: `Warning:[MetadataCommand.ExtractMetadata](../src/core/Akka.Streams/Util/EnumeratorEnumerable.cs#L66)Could not resolve base documentation for 'Akka.Streams.Util.ContinuallyEnumerable`1.ContinuallyEnumerator.MoveNext'`

Reply:
> Yes but the ‘exception’ tag contains useful information - did that throw or just inheritdoc?
> […](#)
> Sent from my iPhone
> On Feb 8, 2022, at 7:56 AM, Ebere Abanonu ***@***.***> wrote: ﻿ @eaba commented on this pull request. In src/core/Akka.Streams/Util/EnumeratorEnumerable.cs: > @@ -60,10 +60,7 @@ public ContinuallyEnumerator(Func<IEnumerator<T>> enumeratorFactory) public void Dispose() => _enumeratorFactory = null; - /// <inheritdoc/> - /// <exception cref="ArgumentException"> DocFx: Warning:[MetadataCommand.ExtractMetadata](../src/core/Akka.Streams/Util/EnumeratorEnumerable.cs#L66)Could not resolve base documentation for 'Akka.Streams.Util.ContinuallyEnumerable1.ContinuallyEnumerator.MoveNext'` — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.

I think `inheritdoc` is the cause, can I delete only the `inheritdoc` and retain the exception?

Reply:
Yes you can I believe